tests/molecule/resources/monitoring/inventory/hosts.yml

```diff
@@ -0,0 +1,39 @@
+---
+all:
+  # List of all servers, defining their alias and IP (hostname)
+  hosts:
+    mclient:
+      hostname: molecule.mclient.local
+      ansible_ip: 192.168.56.2
+      ansible_port: 22
+
+    mserv:
+      hostname: molecule.mserv.local
+      ansible_ip: 192.168.56.3
+      ansible_port: 22
+
+  # Ansible groups. Groups allow configuration and variables to be shared between hosts
+  # Variables in group_vars/all will be shared between all hosts
+  children:
+    # All the monitoring service (docker) servers. Variables in group_vars/monitoring_server will be shared
+    # between these hosts
+    monitoring_server:
```

Comment:
could we rename this to `monitoring_host` to be consistent with the container service in `misrg.xnat`?

Reply:
Changed as suggested.